### PR TITLE
Mk 665 taquito package readme updates

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Ensure that each package in Taquito has a readme and that the readmes have the basic boilerplate for each file.

Devs will need to review after this to make corrections and to add important additional content for users.

We will need to follow up on 665.

- [x ] Your code builds cleanly without any errors or warnings
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
need someone still to review the readmes and add additional dev content

- [ ] You have added or updated corresponding documentation

### CODE REVIEW

File: packages/taquito-tzip16/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Taquito TZip-16 package
+
+`@taquito/tezbridge-tzip16` is an npm package that provides developers with TZip-16 functionality for Taquito. It can be injected as follows:
```
Comment (roxaneletourneau) [inline packages/taquito-tzip16/README.md:3]: provides the ability to retrieve metadata associated with a smart contract based on the TZIP-16 standard. The package also provides a way to execute the MichelsonStorageView found in the metadata

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: packages/taquito-http-utils/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Taquito HTTP Utilities package
+
+`@taquito/http-utils` is an npm package that provides developers with Beacon wallet functionality for Taquito. It can be injected as follows:
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply packages/taquito-http-utils/README.md:3]: whoops thanks! fixed.

File: packages/taquito-signer/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Taquito Signer package
+
+`@taquito/signer` is an npm package that provides developers with signing functionality for Taquito. It can be injected as follows:
+
+```ts
+import { TezosToolkit } from '@taquito/taquito'
+import { Signer } from '@taquito/signer'
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply packages/taquito-signer/README.md:7]: fixed.

File: packages/taquito-tezbridge-signer/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Taquito Tezbridge Signer package
+
+`@taquito/tezbridge-signer` is an npm package that provides developers with ledger signing functionality for Taquito. It can be injected as follows:
+
+```ts
+import { TezosToolkit } from '@taquito/taquito'
+import { tezbridgeSigner } from '@taquito/tezbridge-signer'
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply packages/taquito-tezbridge-signer/README.md:7]: fixed

File: packages/taquito-tezbridge-wallet/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Taquito Tezbridge Wallet package
+
+`@taquito/tezbridge-wallet` is an npm package that provides developers with ledger signing functionality for Taquito. It can be injected as follows:
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply packages/taquito-tezbridge-wallet/README.md:3]: fixed

File: packages/taquito-tzip12/README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Taquito Tzip-12 package
+
+`@taquito/tzip12` is an npm package that provides developers with ledger signing functionality for Taquito. It can be injected as follows:
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply packages/taquito-tzip12/README.md:3]: fixed

File: packages/taquito-tzip12/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+```ts
+import { TezosToolkit } from '@taquito/taquito'
+import { TZip12 } from '@taquito/tzip12'
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply packages/taquito-tzip12/README.md:7]: added

File: packages/taquito-tzip16/README.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+```ts
+import { TezosToolkit } from '@taquito/taquito'
+import { TZip16 } from '@taquito/tzip16'
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply packages/taquito-tzip16/README.md:7]: added

File: packages/taquito-tzip16/README.md (hunk 1 above)
Comment (michaelkernaghan) [reply packages/taquito-tzip16/README.md:3]: added

**Review by michaelkernaghan — COMMENTED** [review]
beacon wallet package: line 3: "given"  should be "giving"

File: packages/taquito-signer/README.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+const Tezos = new TezosToolkit('https://YOUR_PREFERRED_RPC_URL');
+Tezos.setProvider({
+  signer: await InMemorySigner.fromSecretKey('your_private_key', 'your_passphrase'),
+});
+```
+
+### Using a testnet faucet key
+
+To load a faucet key (available from https://faucet.tzalpha.net/) for working a public testnet use the `importKey` function.
+can do so as follows:
```
Comment (michaelkernaghan) [inline packages/taquito-signer/README.md:38]: "can do so as follows:" -> "You can do so as follows:"

File: packages/taquito-tezbridge-signer/README.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
+# Taquito Tezbridge Signer package
+
+`@taquito/tezbridge-signer` is an npm package that provides developers with Tezbridge Signer functionality for Taquito. 
+
+You first need to include https://www.tezbridge.com/plugin.js in your application to use this signer
```
Comment (michaelkernaghan) [inline packages/taquito-tezbridge-signer/README.md:5]: add period to end of the sentence

File: packages/taquito-tzip12/README.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
+# Taquito Tzip-12 package
+
+`@taquito/tzip12` is an npm package that provides developers with Tzip-12 functionality for Taquito. The package allows retrieving metadata associated with tokens of FA2 contract. You can find more information about the TZIP-12 standard [here](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-12/tzip-12.md).
```
Comment (michaelkernaghan) [inline packages/taquito-tzip12/README.md:3]: "FA contract"  -> "FA contracts"

**Review by michaelkernaghan — COMMENTED** [review]
just a few minor typos.

File: packages/taquito-ledger-signer/README.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
+# Taquito Ledger Signer package
+
+`@taquito/ledger-signer` is an npm package that provides developers with ledger signing functionality for Taquito. It implements the Signer interface of Taquito, allowing you to sign operation from a Ledger Nano device.
```
[the remark this answers is not in the document]
Comment (roxaneletourneau) [reply packages/taquito-ledger-signer/README.md:3]: Fixed, thanks!

File: packages/taquito-ledger-signer/README.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+You first need to import the desired transport from the [LedgerJs library](https://github.com/LedgerHQ/ledgerjs). The Ledger Signer has currently been tested with `@ledgerhq/hw-transport-node-hid` for Node-based application and with `@ledgerhq/hw-transport-u2f` for web applications.
```
[the remark this answers is not in the document]
Comment (roxaneletourneau) [reply packages/taquito-ledger-signer/README.md:5]: Fixed, thanks!

File: packages/taquito-local-forging/README.md [hunk 14]
```diff
@@ -1,12 +1,13 @@
-[![pkgsign status](https://us-central1-pkgsign.cloudfunctions.net/pkgsign-badge?name=@taquito/local-forging&expectedIdentity=%40jevonearth)](https://github.com/RedpointGames/pkgsign)
 
 # Taquito Local forging package
 
 `@taquito/local-forging` is an npm package that provides developers with local forging functionality for Taquito. It can be injected as follow.
```
[the remark this answers is not in the document]
Comment (roxaneletourneau) [reply packages/taquito-local-forging/README.md:4]: Fixed, thanks!

File: packages/taquito-signer/README.md (hunk 9 above)
Comment (roxaneletourneau) [reply packages/taquito-signer/README.md:38]: Fixed, thanks!

File: packages/taquito-tzip12/README.md (hunk 11 above)
Comment (roxaneletourneau) [reply packages/taquito-tzip12/README.md:3]: Fixed, thanks!

**Pull request merged**